low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -107,7 +107,7 @@ def copy_input_config(self, stage=None):
         if not os.path.exists(path) or stage == "post_install":
             with open(path, "w") as f:
                 f.write("")
-        Conf.load(GLOBAL_CONFIG_INDEX, global_config_path)
+        Conf.load(GLOBAL_CONFIG_INDEX, global_config_path, skip_reload=True)
```

Comment:
why skipping reload ? This will give away the advantage for which all conf calls we redirected through Conf lib apis !?

Be careful with changes made in sspl setup / mini provisioner interface scripts

Reply:
changed to `fail_reload=False`.